packages/cudos-cmd/commands.js

```diff
@@ -51,10 +40,62 @@ const unitTestInfo = {
   handler: unitTestCmd
 }
 
+const runInfo = {
+  command: 'run <scriptFilePath>',
+  describe: 'run script',
+  builder: (yargs) => {
+    yargs.positional('scriptFilePath', {
+      type: 'string',
+      describe: 'The path to to the script to run'
+    })
+  },
+  handler: runCmd
+}
+
+const keysInfo = {
+  command: 'keys',
+  describe: 'Manage accounts/keys',
+  builder: (yargs) => {
+    yargs.command('ls', 'List all accounts in the node key storage', () => {}, keys.keysListCmd)
+      .command('add <name>', 'Add account to the node key storage', () => {
+        yargs.positional('name', {
+          type: 'string',
+          describe: 'account name'
+        })
+      }, keys.keysAddCmd)
+      .command('rm <name>', 'Remove account from the node key storage', () => {
+        yargs.positional('name', {
+          type: 'string',
+          describe: 'account name'
```

Comment:
why the description of the `add` and `rm` is the same?

Reply:
`account name` is the description of the positional parameter. If you prefer I can change it to a little more descriptive one like `name of the account` or `name of the account to be removed`